src/App.js

```diff
@@ -1,7 +1,9 @@
+import Locations from "./components/LocationsList";
+
 function App() {
```

Comment:
Here you used the function syntax (with `function` keyword), but everywhere else you used arrow functions. It's probably best if you could change this into an arrow function too for consistency.

Reply:
I'll change it with next push. Also some of the functions in the whole applications will need different declaration methods.